src/version-github.ts

```diff
@@ -1,12 +1,11 @@
 import { BaseVersioner } from './version-base';
 import { Octokit } from '@octokit/rest';
-import dotenv from 'dotenv';
-
-dotenv.config();
 
 interface GitHubVersionerOptions {
   owner: string;
   repo: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- OctokitOptions.auth is `any`
+  authOptions?: any;
```

Comment:
Does making this type `OctokitOptions['auth']` avoid the lint? If so, does using that type have any drawbacks? (e.g. does octokit get pulled into the compiled type defs? is that a deal breaker?)

Reply:
I was having a lot of trouble wrangling out OctokitOptions because I couldn't find the correct export and it was a bit of a maze. Since it was `any` I figured this would be simpler.